resolver/src/main/java/io/netty/resolver/HostsFileEntries.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2017 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package io.netty.resolver;
+
+import io.netty.util.internal.UnstableApi;
+
+import java.net.Inet4Address;
+import java.net.Inet6Address;
+import java.util.Collections;
+import java.util.Map;
+
+/**
+ * A container of hosts file entries
+ */
+@UnstableApi
+public class HostsFileEntries {
+
+    /**
+     * Empty entries
+     */
+    public static final HostsFileEntries EMPTY =
```

Comment:
Is really needed caching empty entries? How many HostsFileEntries can be created?